### ISSUE TITLE
Presenting a template and then popping it crashes the app

### ISSUE THREAD

**Issue by tomdye** [birkir/react-native-carplay#27]
When attempting to pop the current template with code similar the what is shown below, the entire application crashes.
This same issue is present in the example app within this repo.

Example:

```ts
useEffect(() => {
    const grid = new GridTemplate({
      buttons: [
        { id: 'foo', titleVariants: ['foo'], image: require('@images/map-pin-small.png') }
      ],
      onButtonPressed: (e) => {
        const alertTemplate = new AlertTemplate({
            titleVariants: ['alert test'],
            backButton: true, // this doesn't do anything?
            actions: [
              { id: 'bar', title: 'bar' }
            ],
            onActionButtonPressed: (id) => {
			  	// This crashes the app
				CarPlay.popTemplate(true);
            }
          });

        CarPlay.presentTemplate(alertTemplate);        
      }
})
```

**Comment by tomdye** [comment]
I've managed to get this working with `dismissTemplate()` but calling `popTemplate()` for other templates still crashes

**Comment by tomdye** [comment]
Will be fixed by https://github.com/birkir/react-native-carplay/pull/33

**Comment by birkir** [comment]
Please re-open if still a problem. Should be fixed with 2.0.0
